flow/FastAlloc.cpp

```diff
@@ -511,7 +511,7 @@ void FastAllocator<Size>::getMagazine() {
 	// FIXME: We should be able to allocate larger magazine sizes here if we
 	// detect that the underlying system supports hugepages.  Using hugepages
 	// with smaller-than-2MiB magazine sizes strands memory.  See issue #909.
-#if !DEBUG_DETERMINISM
+#if 0
 	if (FLOW_KNOBS && g_allocation_tracing_disabled == 0 &&
 	    nondeterministicRandom()->random01() < (magazine_size * Size) / FLOW_KNOBS->FAST_ALLOC_LOGGING_BYTES) {
 		++g_allocation_tracing_disabled;
```

Comment:
I wonder if there's some usage of g_allocation_tracing_disabled that changed recently and made the unseed mismatches manifest

Reply:
It doesn't look like it